### PR TITLE
Setup controllers

### PR DESCRIPTION
**Pull request by souad988**
For each URL (http://127.0.0.1:3000/users/  ; /users/2 ; /users/2/posts ; /users/2/posts/1)created:

- A route.
- An action in the correct controller.
- A view file.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -3,4 +3,9 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  #root 'users#index'
+  resources :users do
+    resources :posts
+  end
```
Comment (mukhammadrizooff) [inline config/routes.rb]: - [ ] [REQUIRED] Good job so far! Should  have `users` and `posts` routes added to `config/routes.rb` and maintained a `RESTful URL` as `:Index | :show` 🙏🏼 

```ruby
# config/routes.rb

  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+class UsersController < ApplicationController
+  def show
+    @user = User.find(params[:id])
+  end
+
+  def posts
+    @user = User.find(params[:id])
```
Comment (mukhammadrizooff) [inline app/controllers/users_controller.rb:11]: - [ ] [REQUIRED] Good job so far! The `users_controller.rb` should contains with `#index` and `#show` actions, please change the `posts` 🙏🏼 

```ruby
class PostsController < ApplicationController
  
  def index; end

  def show; end

end
```

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @souad988, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter errors ✔️ 
- Correct GitFlow ✔️ 
- Good implementations ✔️
- Descriptive `README` file ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/users_controller.rb (hunk 2 above)
Comment (Abenezer-Tilahun) [reply app/controllers/users_controller.rb:11]: Fixed.

File: config/routes.rb (hunk 1 above)
Comment (Abenezer-Tilahun) [reply config/routes.rb]: Fixed.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @souad988 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Git-Flow.
✔️ Professional README  
✔️ Good Commit Messages
✔️ Good job for Implemented Suggested changes.


## Optional suggestions 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

**Pull request merged**